szcc

```diff
@@ -144,15 +169,17 @@ def init():
         verbose = True
 
     STABILIZER_HOME = os.path.dirname(__file__)
-    envpath = os.environ['PATH']
+    envlibpath = os.environ['LD_LIBRARY_PATH']
```

Comment:
Yeah, that won't work.

Reply:
Not sure how else to find the correct path to `libstabilizer.so`.
We could do a check of local dir, then` /usr/lib:/lib:/usr/local/lib` etc, but that gets complicated because can be 32bit and 64bit libraries, and some distros are not consistent in using `lib64` for 64bit libraries, sometimes using a `lib` folder instead
We need it for the -L parameter, but perhaps that is only needed if it is in the local dir instead of a system-controlled lib* dir?

Reply:
Removed this commit entirely.